router/router.go

```diff
@@ -171,6 +172,14 @@ func Load(middleware ...gin.HandlerFunc) http.Handler {
 		debugger.GET("/pprof/trace", debug.TraceHandler())
 	}
 
+	monitor := e.Group("/api/metrics")
+	{
+		monitor.GET("/prometheus",
+			session.MustAdmin(),
```

Comment:
Understood, how about providing a second unrestricted socket for debug / metrics output? Users could then decide if they want to have an authentication mechanism for it or just have it listen on localhost instead. Similar to what docker registry does https://docs.docker.com/registry/configuration/#debug